eox_nelp/pearson_vue/rti_backend.py

```diff
@@ -53,15 +78,17 @@ class AbstractBackend(ABC):
         handle_error(exception: Exception, failed_step_pipeline: str): Handles errors during pipeline execution.
     """
 
-    def __init__(self, **kwargs):
+    def __init__(self, use_audit_pipeline=True, **kwargs):
```

Comment:
The advantage of this is that you can change the backend behavior based on the initialization, but  do we need that?, it's not better stable backends whose execution or behavior is always the same ?

```suggestion
    use_audit_pipeline=True
    
    def __init__(self, **kwargs):
```

Reply:
Yes, but then I must apply the decorator to the specific pipeline backend. Otherwise I would get something like this.
https://edunext.stage.nelp.gov.sa/admin/eox_audit_model/auditmodel/32026/change/

Reply:
You can see the error pipeline is also audited.

Reply:
what happen if you override that in the subclass ?